packages/ckeditor5-ui/theme/globals/_evaluationbadge.css

```diff
@@ -4,8 +4,9 @@
  */
 
 :root {
-	--ck-evaluation-badge-font-size: 7.5px;
-	--ck-evaluation-badge-line-height: 7.5px;
+	--ck-evaluation-badge-font-size: 0.576923em;
+	--ck-evaluation-badge-line-height: 0.576923em;
+	--ck-evaluation-badge-letter-spacing: -0.015em;
 	--ck-evaluation-badge-padding-vertical: 2px;
```

Comment:
Maybe we could order those CSS variables similarly to the CSS properties? Like padding at the top and so on.